libs/ngrx-toolkit/src/lib/with-devtools/with-devtools.ts

```diff
@@ -87,7 +88,8 @@ export function withDevtools<Input extends EmptyFeatureResult>(
 ): SignalStoreFeature<Input, EmptyFeatureResult> {
   return (store) => {
     const isServer = isPlatformServer(inject(PLATFORM_ID));
-    if (isServer) {
+    const { logOnly } = inject(DEVTOOLS_CONFIG, { optional: true }) || DEFAULT_DEVTOOLS_CONFIG;
+    if (isServer || logOnly) {
```

Comment:
`logOnly` doesn't have an effect at the moment. Logging still happens but features like time travelling will be disabled.

Since we "only" have logging at the moment, we can leave this condition with `isServer` only.

Reply:
I don't know if we should include this config for now. Maybe let's stash it for later (when it will do something) and merge only the tree-shaking docs ?

Reply:
I am fine with merging only the docs